compiler/luci-codegen/README.md

```diff
@@ -0,0 +1,203 @@
+# luci-codegen
+
+## Overview
+
+`luci-codegen` component is responcible for native code generation from graph representation.
+
+Package provides:
+- Library `luci_codegen`: takes luci IR, replaces some nodes with special "Compiled" custom nodes.
```

Comment:
In my opinion, it is quite tricky to understand difference between `luci_codegen` and `circle_codegen`. Is it possible to rename one of them?

Reply:
I renamed `circle_codegen` to `native_compiler`, hope it is better now